Simplenote/Classes/UIFont+Simplenote.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+import UIKit
+
+
+// MARK: - UIFont Simplenote Helpers
+//
+extension UIFont {
+
+    /// Returns the System Font for a given Style and Weight
+    ///
+    static func preferredFont(for style: TextStyle, weight: Weight) -> UIFont {
```

Comment:
So this will be called once for every tag cell, and once for every header view, anytime style is refreshed, or when searching. Does that sound right? 
Not sure how expensive a call it is to redo this work over and over.  Only 5 tag cells so maybe not very. Would it be worth caching the response for a particular style and weight?

Reply:
I **love** this idea. Working on it!! Thank you!